### ISSUE TITLE
Nothing wrong except slow loading from map to map

### ISSUE THREAD

**Issue by blarter** [masterfeizz/Xash3DS#52]
whenever I walk into a new map it takes between 20 seconds to 5 minutes to load a map.

**Comment by Millymilo00** [comment]
This isn't anything the dev can really fix, this is due to the sd cards slow read speed.

**Comment by blarter** [comment]
Hmm can I move the files to the device itself?

**Comment by Millymilo00** [comment]
I don't think you can do that but even if you could you'd have to edit this program to look for the files in the 3ds nand.
